test/builtin_dataset_mocks.py

```diff
@@ -728,32 +730,33 @@ def _make_detection_anns_folder(cls, root, name, *, file_name_fn, num_examples):
     def _make_detection_ann_file(cls, root, name):
         def add_child(parent, name, text=None):
             child = ET.SubElement(parent, name)
-            child.text = text
+            child.text = str(text)
             return child
 
         def add_name(obj, name="dog"):
             add_child(obj, "name", name)
-            return name
 
-        def add_bndbox(obj, bndbox=None):
-            if bndbox is None:
-                bndbox = {"xmin": "1", "xmax": "2", "ymin": "3", "ymax": "4"}
+        def add_size(obj):
+            obj = add_child(obj, "size")
+            size = {"width": 0, "height": 0, "depth": 3}
```

Comment:
VOC provides the image size together with the annotations. Since the reworked `BoundingBox` requires the image size, we need to add it to the mock data.